priv/static/specs/http_api_spec.yaml

```diff
@@ -55,6 +55,25 @@ paths:
   /contexts:
     delete:
       summary: 'delete all contexts'
+      parameters:
+        - name: version
+          in: query
+          required: false
+          type: string
+          description: 'Version'
+          default: 'v1'
+        - name: imsi
+          in: query
+          required: false
+          type: integer
+          description: 'International Mobile Subscriber Identity'
+          default: '111111111111111'
+        - name: upf-fqdn
+          in: query
+          required: false
+          type: string
+          description: 'UPF FQDN'
```

Comment:
the implementation actually treats upf-fqdn as regex. Might be worth mentioning here.